proxy/proxy_contract.go

```diff
@@ -23,12 +23,12 @@ func init() {
 
 func OnNEP17Payment(from interop.Hash160, amount int, data interface{}) {
 	caller := runtime.GetCallingScriptHash()
-	if !common.BytesEqual(caller, []byte(gas.Hash)) {
+	if !common.BytesEqual(caller, interop.Hash160(gas.Hash)) {
 		panic("onNEP17Payment: alphabet contract accepts GAS only")
```

Comment:
Let's fix this panic message typo in this PR with extra commit: `onNEP17Payment: proxy contract accepts GAS only`.

Reply:
attached additional commit to this PR